### ISSUE TITLE
Image annotation: example output is broken

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#2563]
This doesn't seem right:
![image](https://github.com/user-attachments/assets/4339ca6d-7c5b-4ee5-a17c-2055e931d495)

Location https://markbind.org/userGuide/components/imagesAndDiagrams.html#annotations

Not sure if the feature itself is broken, or it's the example that is broken (due to how it is generated in the UG).

**Comment by tlylt** [comment]
Yup there is indeed a bug for the annotation feature. Checking on the history of commits, it was alright in this PR: https://github.com/MarkBind/markbind/pull/2428
- https://deploy-preview-2428--markbind-master.netlify.app/userguide/components/imagesanddiagrams
- ![image](https://github.com/user-attachments/assets/4843040f-ee96-465f-a446-12118fd8194e)


But started to have issues when https://github.com/MarkBind/markbind/pull/2427 is merged.
- https://deploy-preview-2427--markbind-master.netlify.app/userguide/components/imagesanddiagrams
- ![image](https://github.com/user-attachments/assets/089819d2-5239-4bf2-b395-8f130dac5bce)
- FYI @yucheng11122017
